hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/AirdropHandlerHelper.java

```diff
@@ -304,4 +318,23 @@ public static PendingAirdropRecord createPendingAirdropRecord(
                 .pendingAirdropValue(pendingAirdropValue)
                 .build();
     }
+
+    /**
+     * Validate if AccountId is a system account.
+     *
+     * @param accountID the ID of the account that need to be validated
+     * @return boolean value indicating if the account is a system account
+     */
+    public static boolean validateIfSystemAccount(@NonNull AccountID accountID) {
+        requireNonNull(accountID);
+        if (isAlias(accountID)) {
+            return false;
```

Comment:
Is this accurate ? Can we not have long zero form of an account number here ?

Reply:
Hm yes, that might not be correct, we have to figure how to check if the account is system account, when we provide alias, any ideas Neeha?

Reply:
we could just get `Account` object from a store and look at its number ?